ui/src/Websocket.js

```diff
@@ -0,0 +1,79 @@
+var wss = new Object();
+
+MF.log = function(msg) {
+    console.log(msg);
+    app.ports.websocketIn.send(msg);
+}
+
+MF.url = function(data) {
+    return 'wss://localhost/ws/channels/' + data.channelid + '/messages?authorization=' + data.thingkey
+}
+
+app.ports.connectWebsocket.subscribe(function(data) {
+    var url = MF.url(data);
+    if (wss[url]) {
+        MF.log("Websocket already open. URL: " + url );
```

Comment:
Can you use `''` instead of `""` here and in the rest of the code?

Reply:
The idea was to use single quotes for the things that happen under the hood and to use the double quotes for the messages intended for the user.